libs/ui-lib-tests/cypress/integration/use-cases/create-cluster/with-external-partner-integrations.cy.ts

```diff
@@ -113,6 +113,11 @@ describe('Create a new cluster with external partner integrations', () => {
         .findDropdownToggle()
         .should('be.disabled');
     });
+    it('Validate that for OCP version 4.15 we show Technology Preview Badge in OCI option', () => {
+      ClusterDetailsForm.openshiftVersionField.selectVersion('4.15');
+      ClusterDetailsForm.externalPartnerIntegrationsField.findDropdown().click();
+      cy.get(`[data-testid="EXTERNAL_PLATFORM_OCI-support-level"]`).should('exist');
```

Comment:
This data-testid does not imply the badge is for Technology preview, it could be for Developer preview too.

You probably need to use `findDropdownItem` and check the text content for `Technology preview`